scripts/create_manifest.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import json
+import os
+
+lockfile = {}
+
+with open("Pipfile.lock") as json_file:
+    lockfile = json.load(json_file)
+
+with open("koku-manifest", "w") as manifest:
+    for name, value in sorted(lockfile["default"].items()):
+        if "version" in value:
+            version = value["version"].replace("=", "")
+            manifest.write("koku/python:3.6=%s:%s\n" % (name, version))
```

Comment:
In case we do upgrade the Python version what do you think about grabbing the Python version this way 

```
>>> import sys
>>> sys.version
'3.7.4 (default, Sep  7 2019, 18:27:02) \n[Clang 10.0.1 (clang-1001.0.46.4)]'
>>> sys.version.split(' ')[0]
'3.7.4'
```

Reply:
Yeah, I thought about doing it programmatically, but since this is run by the developer (who right now could be running 3.7), I thought It should be a more intentional choice. But I'm flexible; I can change it if you prefer. Or I could look into some mechanism where this is generated from within our docker image; but that may be slow and heavy handed.

Reply:
+1 to using `sys.version`.

Reply:
@chambridge That's a good point. Since my example _clearly_ had 3.7. Turns out you can't brew install anything these days without it auto-updating your Python. I'll have to be careful here...

Reply:
@blentz @adberglund Updated to programmatically obtain python version

Reply:
This actually causes a problem in travis when the 3.7 version runs as it detects a difference in the manifest since it runs the create manifest and compares it:
```
$ .travis/check_manifest.sh
Pipfile.lock changed without updating koku-manifest. Run 'make manifest' to update.
```
I'm going to move back to us having to update this when we change the running python.